code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -5112,6 +5112,217 @@
+	color = "#664300"
+
+/datum/reagent/ethanol/waifu/on_mob_life(var/mob/living/M)
+	if(..())
+		return 1
+	if(M.gender == MALE)
+		M.setGender(FEMALE)
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		if(!M.is_wearing_item(/obj/item/clothing/under/schoolgirl))
+			var/turf/T = get_turf(H)
+			T.turf_animation('icons/effects/96x96.dmi',"beamin",-32,0,MOB_LAYER+1,'sound/effects/rejuvinate.ogg',anim_plane = MOB_PLANE)
+			H.visible_message("<span class='warning'>[H] dons her magical girl outfit in a burst of light!</span>")
+			var/obj/item/clothing/under/schoolgirl/S = new /obj/item/clothing/under/schoolgirl(get_turf(H))
+			if(!H.put_in_hands(S) && H.w_uniform) //For quickswap, it needs to be in a hand for some reason. But...
+				H.w_uniform.forceMove(get_turf(H)) //If both hands are occupied, we have another solution: just drop the current suit (empties ID and pockets)
+				H.w_uniform.unequipped()
+			H.equip_to_slot_if_possible(S, slot_w_uniform, 1, 1, 1) //This is the safe quickswap proc.
+			holder.remove_reagent(WAIFU,4) //Generating clothes costs extra reagent
+	M.regenerate_icons()
+
+/datum/reagent/ethanol/scientists_serendipity
+	name = "Scientist's Serendipity"
+	id = SCIENTISTS_SERENDIPITY
+	description = "Go ahead and blow the research budget on drinking this." //Can deconstruct a glass with this for loadsoftech
+	color = "#664300"
+	custom_metabolism = 0.01
+	dupeable = FALSE
+
+/datum/reagent/ethanol/beepskyclassic
+	name = "Beepsky Classic"
+	id = BEEPSKY_CLASSIC
+	description = "Some believe that the more modern Beepsky Smash was introduced to make this drink more popular."
+	color = "#664300" //rgb: 102, 67, 0
+	custom_metabolism = 2 //Ten times the normal rate.
+
+/datum/reagent/ethanol/beepskyclassic/on_mob_life(var/mob/living/M)
+	if(..())
+		return 1
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		if(H.job in list("Security Officer", "Head of Security", "Detective", "Warden"))
+			playsound(get_turf(H), 'sound/voice/halt.ogg', 100, 1, 0)
+		else
+			H.Knockdown(10)
+			playsound(get_turf(H), 'sound/weapons/Egloves.ogg', 100, 1, -1)
+
+/datum/reagent/ethanol/spiders
+	name = "Spiders"
+	id = SPIDERS
+	description = "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA."
+	color = "#666666" //rgb(102, 102, 102)
+	custom_metabolism = 0.01 //Spiders really 'hang around'
+
+/datum/reagent/ethanol/spiders/on_mob_life(var/mob/living/M)
+	if(..())
+		return 1
+	M.take_organ_damage(REM, 0) //Drinking a glass of live spiders is bad for you.
+	if(holder.get_reagent_amount(SPIDERS)>=4) //The main reason we need to have a minimum cost rather than just high custom metabolism is so that someone can't give themselves an IV of spiders for "fun"
+		var/calculate_offset = locate(M.x + pick(1,-1), M.y, M.z)
+		new /mob/living/simple_animal/hostile/giant_spider/spiderling(calculate_offset)
+		holder.remove_reagent(SPIDERS,4)
+		M.emote("scream", , , 1)
```

Comment:
I think this may need redoing after the emote rework.

Reply:
Worked fine when I tested it for a minute.